website/docusaurus.config.js

```diff
@@ -46,6 +46,7 @@ module.exports = {
   tagline: 'Build optimized websites quickly, focus on your content',
   organizationName: 'facebook',
   projectName: 'docusaurus',
+  noIndex: false,
```

Comment:
due to default value  being false in schema, this shouldn't be needed

Reply:
Done